server/test/lib/scene/scene.checkTrigger.test.js

```diff
@@ -446,6 +446,41 @@ describe('scene.checkTrigger', () => {
     });
   });
 
+  it('should execute scene with system start trigger', async () => {
+    sceneManager.addScene({
+      selector: 'my-scene',
+      active: true,
+      actions: [
+        [
+          {
+            type: ACTIONS.LIGHT.TURN_OFF,
+            devices: ['light-1'],
+          },
+        ],
+      ],
+      triggers: [
+        {
+          type: EVENTS.SYSTEM.START,
+          area: 'area-1',
+          user: 'tony',
```

Comment:
Those attributes are not linked to the trigger

Reply:
Oops bad copy/paste